db/migrate/20200929142921_add_ct_columns_to_lighthouse_samples.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# Adds the CT channel columns to the lighthouse_sample table
+class AddCtColumnsToLighthouseSamples < ActiveRecord::Migration
```

Comment:
Think we should try to be consistent with naming Ct VS Cq? I just googled it and it seemed like they mean the same thing.

Reply:
Was also wondering whether it's worth adding something in the column descriptions about what a 'channel' is? ( I don't know...)